src/palimpzest/core/lib/schemas.py

```diff
@@ -348,6 +348,10 @@ class File(Schema):
     filename = StringField(desc="The UNIX-style name of the file")
     contents = BytesField(desc="The contents of the file")
 
+class TextFile(Schema):
```

Comment:
Curious, why is this needed?

Reply:
The current `File` schema has a bytes field instead of a string field; some legacy demos break without the `TextFile` having a string field, and I think others require having bytes (i.e. they use `File`). So for now we just have a copy of each for backwards compatibility.